example/example/settings/base.py

```diff
@@ -212,18 +212,12 @@
 HEADLESS_PREVIEW_LIVE = True
 
 try:
-    from channels.asgi import get_channel_layer  # noqa
-
-    INSTALLED_APPS += ["channels"]
-    ASGI_APPLICATION = "asgi.channel_layer"
-    CHANNELS_WS_PROTOCOLS = ["graphql-ws"]
-
-    CHANNEL_LAYERS = {
-        "default": {
-            # "BACKEND": "asgi_redis.RedisChannelLayer",
-            "BACKEND": "asgiref.inmemory.ChannelLayer",
-            "ROUTING": "grapple.urls.channel_routing",
-        }
-    }
+    import channels
+
+    INSTALLED_APPS += [
+        "channels",
+    ]
+    ASGI_APPLICATION = "graphql_ws.django.routing.application"
```

Comment:
The README/docs need updating to reflect this too

Reply:
I updated these in the preview section as it is at the moment
- https://wagtail-grapple.readthedocs.io/en/latest/getting-started/installation.html#installation 
- https://wagtail-grapple.readthedocs.io/en/latest/general-usage/preview.html

Should I add a section in the installation section ?
If you think I should, it might be better to have a dedicated section about "How to enable channels" because it might be a bit confusing to mix default installation steps with channel-enabled steps.

Reply:
a separate sections sounds great

Reply:
Thanks for your feedback, I will add it tomorrow.